tests/acceptance/features/cliDataExporter/export.feature

```diff
@@ -19,4 +19,47 @@ Feature: An administrator wants to export the files of his user using
     And as "user0" file "trashbinFile.txt" should exist in the trashbin
     When user "user0" is exported to path "/tmp/fooSomething" using the occ command
     Then the last export should contain file "/testFolder/testfile.txt" with content "hello"
-    And the last export should contain file "trashbinFile.txt" with content "file in trash bin" in trashbin
\ No newline at end of file
+    And the last export should contain file "trashbinFile.txt" with content "file in trash bin" in trashbin
+
+  Scenario Outline: Multiple deleted file with almost similar name should be contained in an export
+    Given user "user0" has created folder "testFolder"
+    And user "user0" uploads file with content "hello" to "/testFolder/testfile.txt" using the WebDAV API
+    And user "user0" uploads file with content "first file in trash bin" to "/testFolder/<firstFileName>" using the WebDAV API
+    And user "user0" uploads file with content "second file in trash bin" to "/testFolder/<secondFileName>" using the WebDAV API
```

Comment:
```suggestion
    And user "user0" has uploaded file with content "hello" to "/testFolder/testfile.txt"
    And user "user0" has uploaded file with content "first file in trash bin" to "/testFolder/<firstFileName>"
    And user "user0" has uploaded file with content "second file in trash bin" to "/testFolder/<secondFileName>"
```

Reply:
Or you need to make them when steps instead of using `And` continuously after `Given`